src/neo/SmartContract/CallFlags.cs

```diff
@@ -12,6 +12,8 @@ public enum CallFlags : byte
         AllowCall = 0b00000100,
         AllowNotify = 0b00001000,
 
+        IfExists = 0b10000000,
+
         ReadOnly = AllowStates | AllowCall | AllowNotify,
         All = AllowStates | AllowModifyStates | AllowCall | AllowNotify
```

Comment:
Should `All` include `IfExists`?

Reply:
I don't think so, but maybe we can rename `All` to `Default`

Reply:
Yes, we need it

Reply:
Done